language-ruby_5bb9e082042863158cc7236d.md

```diff
@@ -92,6 +93,34 @@ Semaphore doesn't set language specific environment variables like
             - bundle exec rake test
 </code></pre>
 
+## Rails
+
+Configuring Rails applications require a few extra steps. You'll need
+to set `RAILS_ENV` like discussed earlier and generate a
+`config/database.yml`. The simplest way to start a database with
```

Comment:
I think that the example can be made more clear:
- `DATABASE_ENV` is a special env var that Rails understands, but the example seems to use it as random temporary storage variable
- Rails 4.1+ can read db config from from `DATABASE_ENV` only
- However older versions need `config/database.yml`

I suggest providing two examples, one for 4.1+ with `DATABASE_ENV`, and another with `config/database.yml` which is stored in git and cp-ed in prologue. I suggest we show users to store such Semaphore-specific files in `.semaphore` folder, along with pipeline definition(s).